src/Joomlatools/Console/Command/Site/Configure.php

```diff
@@ -81,6 +94,14 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $this->_promptDetails($input, $output);
         }
 
+        if (null !== ($file = $input->getOption('options'))) {
+        	if (!file_exists($file)) {
+        		throw new \InvalidArgumentException((sprintf("Argument file `%s' not found", $file)));
+	        }
+	        $opt = file_get_contents($file);
+        	$this->_extra_options = unserialize($opt);
```

Comment:
@msaladna Would JSON or YAML also work for you here, instead of a serialized format? That makes it easier to create and edit such a file. 

We could use the symfony/yaml package for this.

Reply:
Added `Deserializer` class that will deserialize the input based upon extension. Supports YAML, JSON, and native.